src/System.Web.WebPages/Helpers/AntiXsrf/AntiForgeryWorker.cs

```diff
@@ -104,7 +104,11 @@ public TagBuilder GetFormInputElement(HttpContextBase httpContext)
                 // Adding X-Frame-Options header to prevent ClickJacking. See
                 // http://tools.ietf.org/html/draft-ietf-websec-x-frame-options-10
                 // for more information.
-                httpContext.Response.AddHeader("X-Frame-Options", "SAMEORIGIN");
+                const string frameHeaderName = "X-Frame-Options";
```

Comment:
Shoot. Command-line build now fails with
```
SA1303: Constants must start with an upper-case letter: frameHeaderName.
```

Please rename this again -- `FrameHeaderName`. (I wasn't aware this error was enabled.)

Reply:
How about now :)